package.json

```diff
@@ -71,43 +71,7 @@
     "yarn"
   ],
   "author": "Rhys Arkins <rhys@arkins.net>",
-  "contributors": [
```

Comment:
Made a empty array out of the contributors in the `package.json`.
I will attempt to migrate nearly all of those once we have the bot properly configured and running.